### PR TITLE
i18n: Event Dashboard

### LINKED ISSUE #76: i18n: Event Dashboard
As a translator I want to be able to localise these pages:
- Event Dashboard - https://wedance.vip/events/2e7DJP7C39iy8ibbEq0w/dashboard

To access Event Dashboard:
- Create event: click "plus" icon in the header of https://wedance.vip/events/ page, fill the fields and click save
- On the event page click more (three dots in the header) and click Dashboard

Follow i18n style guide in #70

### PR DESCRIPTION
**Pull request by harshbrahmxatriya**
How to access Event Dashboard:

- Go to Calendar in Main Navigation
- Create event - click "plus" icon in the header
- Fill the fields and click save
- On the event page click more (three dots in the header) and click Dashboard
    
Closes #76

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -283,3 +283,16 @@ TInputProfile:
   addInstagram: (add from Instagram)
   addFacebook: (add from Facebook)
   placeholder: Search on WeDance, Instagram or Facebook
+
+EventDashboard:
+  main: Only event owner can access this area.
```
Comment (razbakov) [inline locales/en.yml:288]: Give more context to translators. It's not "main" section. It's just an error message that user don't have an access.

```suggestion
  noAccess: Only event owner can access this area.
```

File: locales/en.yml [hunk 2]
```diff
@@ -283,3 +283,16 @@ TInputProfile:
...
+  buttonUnlink: Unlink
+  Couple: Couple
+  Button:
```
Comment (razbakov) [inline locales/en.yml:291]: You can dissolve `Button` and just put all keys one level higher.

File: pages/events/_id/dashboard.vue [hunk 3]
```diff
@@ -5,7 +5,7 @@
     v-else-if="!can('edit', 'posts', item)"
     class="mt-4 mx-auto max-w-md p-4 text-sm text-center"
   >
-    Only event owner can access this area.
+    {{$t("EventDashboard.main")}}
```
Comment (razbakov) [inline pages/events/_id/dashboard.vue:8]: Do you have Prettier extension installed? There are spaces missing. Prettier will format it for you. Just install it.

```suggestion
    {{ $t("EventDashboard.main") }}
```

File: pages/events/_id/dashboard.vue [hunk 4]
```diff
@@ -88,7 +88,7 @@
                     update(item.partnerId, { partnerId: '' })
                     update(item.id, { partnerId: '' })
                   "
-                  >Unlink</TButton
+                  >{{$t("EventDashboard.buttonUnlink  ")}}</TButton
```
Comment (razbakov) [inline pages/events/_id/dashboard.vue:91]: Did you test it? This key won't be found as you have extra spaced here.

```suggestion
                  >{{$t("EventDashboard.buttonUnlink")}}</TButton
```

**Review by razbakov — CHANGES_REQUESTED** [review]
I mentioned corrections in the first place where I saw them, but all my comments apply for all occurrences in code.

File: locales/en.yml [hunk 5]
```diff
@@ -283,3 +283,16 @@ TInputProfile:
...
+    Cancel: Cancel
+    CheckIn: CheckIn
+    RSVP: RSVP
+    Subs: Subscribe
+    Paid: Paid
+    NotPaid: Didn't Pay
+
```
[the remark this answers is not in the document]
Comment (harshbrahmxatriya) [reply locales/en.yml:298]: > don't add new lines in the end of file
But in the event and eventview there is just dashboard: dashboard , so should I add these below the dashboard: dashboard line?

File: locales/en.yml [hunk 6]
```diff
@@ -283,3 +283,16 @@ TInputProfile:
   addInstagram: (add from Instagram)
   addFacebook: (add from Facebook)
   placeholder: Search on WeDance, Instagram or Facebook
+
+EventDashboard:
+  main: Only event owner can access this area.
+  buttonUnlink: Unlink
```
[the remark this answers is not in the document]
Comment (harshbrahmxatriya) [reply locales/en.yml:289]: > No need for extra prefix "button" here. Use camelCase like we do everywhere else, start with small letter.

Alright, noted

File: locales/en.yml (hunk 2 above)
Comment (harshbrahmxatriya) [reply locales/en.yml:291]: > You can dissolve `Button` and just put all keys one level higher.
K

File: pages/events/_id/dashboard.vue (hunk 3 above)
Comment (harshbrahmxatriya) [reply pages/events/_id/dashboard.vue:8]: > Do you have Prettier extension installed? There are spaces missing. Prettier will format it for you. Just install it.

Yes it is installed, now I should format document with it now onwards

File: pages/events/_id/dashboard.vue (hunk 4 above)
Comment (harshbrahmxatriya) [reply pages/events/_id/dashboard.vue:91]: Okay, I'll see to it

File: locales/en.yml [hunk 7]
```diff
@@ -63,6 +63,16 @@ eventView:
...
     edit: Edit
     share: Share
+  EventDashboard:
+    noAccess: Only event owner can access this area.
+    unlink: Unlink
+    couple: Couple
+    cancel: Cancel
+    checkIn: CheckIn
+    rsvp: RSVP
+    subs: Subscribe
```
Comment (razbakov) [inline locales/en.yml:73]: Better to use full words, so that translators can understand the context better. Avoid shortcuts. Same for variables in the code FYI ;)

```suggestion
    subscribe: Subscribe
```

**Review by razbakov — CHANGES_REQUESTED** [review]

**Comment by razbakov** [comment]
@harshbrahmxatriya please rebase onto beta

**Comment by harshbrahmxatriya** [comment]
> @harshbrahmxatriya please rebase onto beta

@razbakov  means it is approved by you?

**Comment by best-lucky1030** [comment]
resolved conflicts.

**Review by best-lucky1030 — COMMENTED** [review]
resolved conflicts

File: components/TInput/TInputArray.vue [hunk 8]
```diff
@@ -31,6 +31,9 @@ export default {
     },
   },
   watch: {
+    internalValue: {
```
Comment (best-lucky1030) [inline components/TInput/TInputArray.vue:34]: why did you add this part?
this was already defined under `value(val)` {}.
I think we have to remove this or if you want, you can remove `old` param from 2 params (val, old).

**Comment by razbakov** [comment]
@best-lucky1030 check slack ;)

**Comment by razbakov** [comment]
@harshbrahmxatriya make sure PR contains only your commits ;)

**Comment by razbakov** [comment]
@harshbrahmxatriya I rebased your branch. Now we can merge it
